README.md

```diff
@@ -0,0 +1,335 @@
+
+We're using `dep` so that way the version of the driver that we're using in our project is version locked.
+
+In addition to the MongoDB Go driver, we're also going to need to get the [AWS Lambda SDK for Go](https://github.com/aws/aws-lambda-go) as well as an [unofficial SDK for Alexa](https://github.com/arienmalec/alexa-go), since no official SDK exists. To do this, we can execute:
+
+```bash
+dep ensure -add "github.com/arienmalec/alexa-go"
+dep ensure -add "github.com/aws/aws-lambda-go/lambda"
+```
+
+With the dependencies available to us, we can modify the project's **main.go** file. Open the file and add the following code:
+
+```go
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type Connection struct {
+	database *mongo.Database
+}
+
+func main() {
+	client, err := mongo.NewClient(options.Client().ApplyURI(os.Getenv("ATLAS_URI")))
+	if err != nil {
+		log.Fatal(err)
+	}
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+	err = client.Connect(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Disconnect(ctx)
+	database := client.Database("alexa")
+	connection := Connection{
+		database: database,
+	}
+}
+```
+
+In the `main` function we are creating a client using the connection string of our cluster. In this case, I'm using an environment variable on my computer that points to my MongoDB Atlas cluster. Feel free to configure that connection string however you feel the most confident.
+
+Before attempting to connect, we are setting a generous timeout of ten seconds. Upon connecting, we are getting a handle of an `alexa` database and storing it in a `Connection` data structure. If the `alexa` database doesn't exist, it will create one if we try to create data.
+
+You can check out more information about connecting to MongoDB with the Go programming language in a [previous tutorial](https://www.mongodb.com/blog/post/quick-start-golang--mongodb--starting-and-setup) I wrote.
+
+So why are we storing the database handle in a `Connection` data structure?
+
+AWS Lambda behaves a little differently when it comes to web applications. Instead of running the `main` function and then remaining alive for as long as your server remains alive, Lambda functions tend to suspend or shutdown when they are not used. For this reason, we cannot rely on our connection being available and we also don't want to establish too many connections to our database in the scenario where our function hasn't shut down. To handle this, we can pass the connection from our `main` function to our logic function.
+
+Let's make a change to see this in action:
+
+```go
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/arienmalec/alexa-go"
+	"github.com/aws/aws-lambda-go/lambda"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type Connection struct {
+	database *mongo.Database
```

Comment:
Instead of storing a handle to a `Database` object that's only used to get handles to `Collection` objects, why not store a handle to the `recipes` collection instead?